site/content/en/docs/technical-outcomes/scheduling/limit-range/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Scenario 4"
+---
+
+# Scenario 4: Scheduling pods with a limit range
+
+A limit range schedule policy can be used in a KWOK cluster.
+
+## Prerequisites
+
+- KWOK must be installed on the machine. See [installation](https://kwok.sigs.k8s.io/docs/user/installation/).
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+
+## Create cluster
+
+```bash
+kwokctl create cluster
+```
+
+## View clusters
+
+This ensures that the cluster was created successfully.
+
+```bash
+kwokctl get clusters
+```
+
+## Deploy node
+
+- Node: 1 worker node in the cluster
+
+{{< expand "node.yaml" >}}
+
+{{< code-sample file="node.yaml" >}}
+
+{{< /expand >}}
+
+```bash
+kubectl apply -f node.yaml
+```
+
+## Create a resource limit
+
+{{< expand "limit-range.yaml" >}}
+
+{{< code-sample file="limit-range.yaml" >}}
+
+{{< /expand >}}
+
+```bash
+kubectl apply -f limit-range.yaml
+```
+
+## Confirm the limit has the required values
+
+```bash
+kubectl describe limitranges cpu-resource-constraint
+
+Name:       cpu-resource-constraint
+Namespace:  default
+Type        Resource  Min   Max  Default Request  Default Limit  Max Limit/Request Ratio
+----        --------  ---   ---  ---------------  -------------  -----------------------
+Container   cpu       100m  1    500m             500m           -
+```
+
+## Deploy a pod above the resource limit
+
+- Pod specification: 0.7 CPU
```

Comment:
```suggestion
- Pod specification: 700m CPU
```

I think we should use a unified CPU unit `m`

Reply:
Ok